Lib/selectors.py

```diff
@@ -446,6 +447,29 @@ class EpollSelector(_PollLikeSelector):
         _EVENT_READ = select.EPOLLIN
         _EVENT_WRITE = select.EPOLLOUT
 
+        def __init__(self):
+            super().__init__()
+            self._exclusive = False
+
+        @property
+        def exclusive(self):
+            return self._exclusive
+
+        def set_exclusive(self, value):
```

Comment:
So far `selectors` haven't had this but looking at `asyncio`, `ssl`, `socket`, `threading`, and `multiprocessing` code, we usually either use properties for *both* getting and setting a value, or we use *both* a `get_*` and a `set_*` method.

Since we already have `get_key` and `get_map` in `selectors.py`, and there are no public fields nor properties here, I'd change the `exclusive` getter to an explicit `get_exclusive()` method.